sopel/config/types.py

```diff
@@ -467,18 +471,20 @@ class SpamSection(StaticSection):
 
         **About:** backward compatibility with comma-separated values.
 
-        A :class:`ListAttribute` option allows to write, on a single line,
-        the values separated by commas. As of Sopel 7.x this behavior is
-        discouraged. It will be deprecated in Sopel 8.x, then removed in
-        Sopel 9.x.
+        A :class:`ListAttribute` option used to allow to write, on a single
+        line, the values separated by commas. It is still technically possible
+        while raising a deprecation warning.
+
+        In Sopel 7.x this behavior was discouraged, as of Sopel 8.x it is now
+        deprecated and raises warnings, and it will be removed in Sopel 9.x.
```

Comment:
```suggestion
        In Sopel 7.x this behavior was discouraged; as of Sopel 8.x it is now
        deprecated with warnings, and it will be removed in Sopel 9.x.
```

Comma splices 💢